ITKImageProcessing/src/ITKImageProcessing/Filters/ITKImportImageStack.cpp

```diff
@@ -241,43 +356,76 @@ Result<> ITKImportImageStack::executeImpl(DataStructure& dataStructure, const Ar
   {
     return MakeErrorResult(-4, fmt::format("Unsupported pixel component: {}", imageIO->GetComponentTypeAsString(component)));
   }
-
+  Result<> readResult;
   switch(*numericType)
```

Comment:
Unrelated to the actual review, but do you want me to but in a template to remove the excess numericType switches? with this filter we have 4 that I could find and presumably more down the line since we have a parameter that uses this enum